.babelrc

```diff
@@ -6,14 +6,30 @@
   "env": {
     "coverage": {
       "plugins": [
-        ["istanbul"]
+        [
+          "flow-react-proptypes",
+          "transform-flow-strip-types",
+          "istanbul"
+        ]
+      ]
+    },
+    "development": {
+      "sourceMaps": "both",
+      "plugins": [
+        "flow-react-proptypes",
```

Comment:
I think that we need that plugin in production too, otherwise people not using flow at left behind.